py/surveyqa/nightly.py

```diff
@@ -151,6 +151,28 @@ def get_nightlytable(exposures):
     return nightly_table
 
 
+def get_moonloc(night):
+    """
+    Returns the location of the moon on the given NIGHT
+    
+    Args:
+        night : night = YEARMMDD of sunset
+    
+    Returns a SkyCoord object
+    """
+    #- Re-formats night into YYYY-MM-DD HH:MM:SS 
+    iso_format = night[:4] + '-' + night[4:6] + '-' + night[6:] + ' 00:00:00'
+    t_midnight = Time(iso_format, format='iso') + 24*u.hour
```

Comment:
Almost.  This is midnight GMT, but we want midnight in Arizona after the sunset on night YEARMMDD.  Midnight in AZ is 7am GMT on the next day, so you are correct about the `+ 24*u.hour`, but we need the time portion of the string to be `07:00:00` not `00:00:00`.  Double check me on that...